packages/office-ui-fabric-react/src/components/Facepile/Facepile.base.tsx

```diff
@@ -90,12 +90,7 @@ export class FacepileBase extends BaseComponent<IFacepileProps, {}> {
         ? onRenderPersona(persona, this._getPersonaControl)
         : onRenderPersonaCoin(persona, this._getPersonaCoinControl);
       return (
-        <li
-          role="option"
```

Comment:
This is being removed because each child of listbox needs to have `role="option"` right? But if we do that aren't we fully removing support of a parent-child relationship with Personas that are used in Facepile?

Reply:
The `<ul />` `<li />` relationship should be sufficient to indicate the parent child relationship between the list and the items in the list. 

Doing this will also make the AT do the native thing with regards to list and announce item 1 or 2 etc. correctly.